plugins/plugins.go

```diff
@@ -365,6 +367,13 @@ func WithTracerProvider(tracerProvider *trace.TracerProvider) func(*Manager) {
 	}
 }
 
+// DistributedTracingOpts sets the options to be used by distributed tracing.
```

Comment:
Nit: `DistributedTracingOpts` -> `WithDistributedTracingOpts`

Reply:
Ah, ok missed that one(the convention differs on the place where it is used. Will change it for consistency